test/e2e/support/navigation.ts

```diff
@@ -134,8 +134,6 @@ const completeFormSlow: { [key in FrontendTestTask]: () => void } = {
       cy.get(appFrontend.grid.bolig.percent).type('80');
       cy.get(appFrontend.grid.studie.percent).type('15');
       cy.get(appFrontend.grid.kredittkort.percent).type('5');
-      cy.get(appFrontend.grid.totalPercent).focus();
-      cy.get(appFrontend.grid.totalPercent).should('have.value', '100 %');
```

Comment:
I removed this because it is tested/asserted elsewhere and doesn't work when you have autoSaveBehavior: "onChangePage" because this value is calculated backend.